garage/misc/git_handler.py

```diff
@@ -0,0 +1,383 @@
+"""Perform different git tasks while running experiments."""
+import importlib
+import signal
+import time
+
+import sh
+from termcolor import colored
+
+from garage.config import GIT_REPO_URL
+from garage.config import PROJECT_PATH
+
+
+def cmd_to_string(command):
+    """Return the output of the executed command as a string.
+
+    Parameter
+    ---------
+    - command: it's an executed command from the sh package.
+
+    """
+    str_cmd = "%s" % command
+    # Remove next line character
+    return str_cmd[:-1]
+
+
+def cmd_to_string_array(command):
+    """Return the output of the executed command as a string array.
+
+    The output is split by the new line character and each element is added to
+    the array.
+
+    Parameter
+    ---------
+    - command: it's an executed command from the sh package.
+
+    """
+    cmd_str = "%s" % command
+    srt_arr = cmd_str.split("\n")
+    # Remove next line character
+    srt_arr = srt_arr[:-1]
+    return srt_arr
```

Comment:
Why not `return str_arr[:-1]` here like function cmd_to_string in line 23? Also, `str_arr` instead of `srt_arr`?

Reply:
Nice catch, done!